roles/lib_utils/tasks/main.yml

```diff
@@ -6,6 +6,8 @@
 
 - name: lib_utils ensure python-ruamel-yaml package is on target
   package:
-    name: python-ruamel-yaml
+    name: "{{ 'python2-ruamel-yaml' if ansible_distribution != 'Fedora' else 'python3-ruamel-yaml' }}"
```

Comment:
I mentioned it in IRC, but didn't specifically mention you.

You could use: `ansible_python.version.major`, which should let you do this:
```
- name: lib_utils ensure python-ruamel-yaml package is installed
  package:
    name: "python{{ ansible_python.version.major }}-ruamel-yaml"
```